app/components/DeleteTodo.js

```diff
@@ -10,6 +10,12 @@ class DeleteTodo extends Component {
         this.state = {}
     }
 
+    function update() {
+      const x = 1;
+      this.update();
+      x++;
+    }
+  
```

Comment:
'refetchQueries' array should be wrapped in square brackets [{...}] and each query should be a separate object. Check the syntax for passing multiple queries.